pom.xml

```diff
@@ -200,7 +200,7 @@
         <jboss-slf4j.version>1.2.1.Final</jboss-slf4j.version>
         <httpclient.version>4.5.13</httpclient.version>
         <apicurio-common-rest-client.version>0.1.12.Final</apicurio-common-rest-client.version>
-        <apicurio-common-app-components.version>0.1.11.Final</apicurio-common-app-components.version>
+        <apicurio-common-app-components.version>0.1.12-SNAPSHOT</apicurio-common-app-components.version>
```

Comment:
Is this expected/desired? Or it is because changes from the app-components are required and we need to wait for a release from the other library?

Reply:
You are right, we are waiting for a release of the other library as this PR have been merged:
https://github.com/Apicurio/apicurio-common-app-components/pull/44
but we have no release for it yet.

Reply:
Fixed now